standard/expressions.md

```diff
@@ -4562,7 +4565,19 @@ The conditional operator is right-associative, meaning that operations are group
 
 The first operand of the `?:` operator shall be an expression that can be implicitly converted to `bool`, or an expression of a type that implements `operator true`. If neither of these requirements is satisfied, a compile-time error occurs.
 
-The second and third operands, `x` and `y`, of the `?:` operator control the type of the conditional expression. If both `x` and `y` are *default_literal*s ([§12.8.20](expressions.md#12820-default-value-expressions)), a compile-time error occurs.
+If `ref` is present:
+Both *expression*s shall be variables having the same type, and that type is the type of the result.
+The result is a variable, which is writeable if both *expression* variables are writeable.
+The run-time processing of a ref conditional expression of the form `b ? ref x : ref y` consists of the following steps:
+
+- First, `b` is evaluated, and the `bool` value of `b` is determined:
+  - If an implicit conversion from the type of `b` to `bool` exists, then this implicit conversion is performed to produce a `bool` value.
+  - Otherwise, the `operator true` defined by the type of `b` is invoked to produce a `bool` value.
+- If the `bool` value produced by the step above is `true`, then `x` is evaluated and the resulting reference becomes the result of the conditional expression.
+- Otherwise, `y` is evaluated and the resulting reference becomes the result of the conditional expression.
+
+If `ref` is not present:
+The second and third operands, `x` and `y`, of the `?:` operator control the type of the conditional expression.
```

Comment:
```suggestion
If `ref` is not present, the second and third operands, `x` and `y`, of the `?:` operator control the type of the conditional expression:
```

Reply:
And I think we're missing the "If both `x` and `y` are *default_literal*s ([§12.8.20](expressions.md#12820-default-value-expressions)), a compile-time error occurs." that we had before.